packages/dapp/src/components/listing/ListingRedux.tsx

```diff
@@ -172,6 +173,12 @@ class ListingPageComponent extends React.Component<
                 </ListingTabContent>
               </Tab>
 
+              <Tab title="Boosts">
```

Comment:
What happens if you wrap `<FeatureFlag>` around this tab?

Reply:
Yes, if I wrap it here the tab isn't clickable and it breaks the styles:

<img width="395" alt="Screen Shot 2019-07-10 at 9 35 35 AM" src="https://user-images.githubusercontent.com/1965433/60973678-b727bc80-a2f6-11e9-9a54-e41479e9d486.png">

Reply:
I'd say let's wait until we enable the feature flag on prod for this.

I wonder if it's fixable though. Here's the line in FeatureFlag: https://github.com/joincivil/Civil/blob/master/packages/components/src/features/FeatureFlag.tsx#L17 - I'm not sure if the `children` prop can be directly output without wrapping it in fragment element, but we could give it a prop that takes a `JSX.Element` and just output that (I'm not sure why we're not doing that with `replacement`).